docs/source/terminology.rst

```diff
@@ -21,30 +21,51 @@ This documents a few common patterns that exist in the Cartographer codebase.
 Frames
 ======
 
-global (map) frame
+global map frame
```

Comment:
Unless you intend to rename the accessors as well, you should mention here that it is sometimes referred to as just global frame or local frame. Either in parens or after a comma for Ctrl+F friendliness.

Reply:
I'm not sure what you would like to be changed. There is no mention of "global frame" or "local frame" in the whole codebase anymore.